megamek/src/megamek/server/Server.java

```diff
@@ -12807,9 +12805,10 @@ else if ((waterDepth > 0)
                     synchronized (cfrPacketQueue) {
                         try {
                             cfrPacketQueue.wait();
-                        } catch (InterruptedException ignored) {
-                            // Do nothing
+                        } catch (Exception ignored) {
+
```

Comment:
Empty catch?

Reply:
Another thread wait, fine to swallow the exception here